tla/ccfraft.tla

```diff
@@ -913,40 +916,42 @@ UpdateCommitIndex(i,j,m) ==
                    votedFor, candidateVars, leaderVars, log, clientRequests, committedLog>>
 
 \* Receive a message.
```

Comment:
```suggestion
\* Receive a message. Used by trace checker.
```
Maybe add a note here so this doesn't get refactored out in the future

Reply:
"The definition  Messages  may be redefined along with  WithMessages  and  WithoutMessages  above.  For example, one might want to model  messages  , i.e., the network as a bag (multiset) instead of a set.  The Traceccfraft.tla spec does this."